fastestimator/op/numpyop/univariate/rua.py

```diff
@@ -189,12 +200,13 @@ def __init__(self,
                  inputs: Union[str, Iterable[str]],
                  outputs: Union[str, Iterable[str]],
                  mode: Union[None, str, Iterable[str]] = None,
+                 ds_id: Union[None, str, Iterable[str]] = None,
                  num_bits: Union[int,
                                  Tuple[int, int],
                                  Tuple[int, int, int],
                                  Tuple[Tuple[int, int], Tuple[int, int], Tuple[int, int]]] = 7):
         self.num_bits = num_bits
-        super().__init__(inputs=inputs, outputs=outputs, mode=mode, num_bits=num_bits)
+        super().__init__(inputs=inputs, outputs=outputs, mode=mode, ds_id=ds_id, num_bits=num_bits)
 
     def set_rua_level(self, magnitude_coef: float) -> None:
         """Set the augmentation intensity based on the magnitude_coef.
```

Comment:
ds_id should be passed to super().__init__() inside the set_rua_level method as well.

Reply:
nice catch